src/app/models/types/serviceConfig.ts

```diff
@@ -165,6 +165,7 @@ export interface Recommendations {
     text: string;
     endTextVisuallyHidden: string;
   };
+  hasStoryRecommendations?: boolean;
```

Comment:
This can be removed from here and from the service config (the functionality has been removed from the application after I had generated the service config template)